server/data/incidentTypeConfiguration/formWizard.ts

```diff
@@ -3,6 +3,8 @@ import FormWizard from 'hmpo-form-wizard'
 import type { AnswerConfiguration, IncidentTypeConfiguration, QuestionConfiguration } from './types'
 import QuestionsController from '../../controllers/wip/questionsController'
 
+const MAX_ANSWERS_PER_PAGE = 20
```

Comment:
i'm astonished that eslint was not upset 🤣 (ps: i like it)

Reply:
eslint should shut it :P